### ISSUE TITLE
Fix bad tel: link in example/index.html 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#40]
```
Run any of the examples.

The double slash should not be after tel:

See 
https://github.com/phonegap/phonegap-android/commit/4f121aa07d21702d2b834ca78afa
bcc092a43e54

```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 5 Sep 2011 at 3:18

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 5 Sep 2011 at 3:19

**Comment by GoogleCodeExporter** [comment]
```
Proposed for 1.2.5
```

Original comment by `paul.beu...@gmail.com` on 20 Oct 2011 at 12:07
- Added labels: **Milestone-Release1.2.5**

**Comment by GoogleCodeExporter** [comment]
```
Fixed and released in 1.2.5
```

Original comment by `paul.beu...@gmail.com` on 1 Nov 2011 at 8:39
- Changed state: **Fixed**
